Packs/RSTCloud/Integrations/RSTCloudThreatFeedAPI/RSTCloudThreatFeedAPI.py

```diff
@@ -0,0 +1,429 @@
+                'Tags': indicator.get('tags', '').get('str', ''),
+                'Threat': indicator.get('threat', ''),
+                'Score': indicator['score'],
+                'Description': string_to_context_key(indicator.get('description', '')),
+                'FalsePositive': indicator.get('fp', '').get('alarm', ''),
+                'FalsePositiveDesc': indicator.get('fp', '').get('descr', '')
+            }
+            dbot_score = {'Indicator': url, 'Type': 'url', 'Vendor': 'RST Cloud',
+                          'Score': calculate_score(total_score, threshold, 'url', int(indicator.get('lseen', '')))}
+            human_readable_score = ''
+            if dbot_score['Score'] == 3:
+                human_readable_score = 'Malicious'
+                outputs[human_readable_score] = {'Vendor': 'RST Cloud',
+                                                 'Description': additional_info['Description'], 'Score': total_score}
+            if dbot_score['Score'] == 2:
+                human_readable_score = 'Suspicious'
+                outputs[human_readable_score] = {'Vendor': 'RST Cloud',
+                                                 'Description': additional_info['Description'], 'Score': total_score}
+            context[outputPaths['url']].append(outputs)
+            context[f'RST.{outputPaths["url"]}'].append(additional_info)
+            context[DBOT_SCORE_KEY].append(dbot_score)
+            table = {'Score': total_score,
+                     'Relevance': human_readable_score,
+                     'Threat': ', '.join(threat for threat in additional_info["Threat"]),
+                     'Last Seen': time.strftime('%Y-%m-%d', time.localtime(int(indicator.get('lseen', '')))),
+                     'Description': f'\n{additional_info["Description"]}\n',
+                     'Tags': ', '.join(tag for tag in additional_info['Tags'])}
+            markdown += tableToMarkdown(f'RST Threat Feed URL Reputation for: {url}\n'
+                                        f'{RST_URL}uuid?id={indicator["id"]}', table, removeNull=True)
+            results.append(indicator)
+
+    return markdown, context, results
+
+
+@logger
+def submit_command(client: Client, args: Dict[str, str]) -> Tuple[str, dict, Any]:
+    iocs = argToList(args.get('ioc', ''))
+    description = argToList(args.get('description', 'manual submission'))
+    context: Dict[str, Any] = defaultdict(list)
+    markdown: str = ''
+    results: List[str] = []
+    num = 0
+    for ioc in iocs:
+        indicator = client.submit_indicator(ioc, description[num])
+        num += 1
+        if "status" in indicator.keys():
+            markdown += f"Indicator: {ioc} was submitted as a potential threat indicator to RST Cloud\n"
+        elif "error" in indicator.keys():
+            markdown += f"Indicator: {ioc} was not submitted successfully due to the following error: {indicator['error']}\n"
```

Comment:
If a command fails, it should raise an exception (e.g. `DemistoException`), not fail gracefully so the playbook will stop on errors. Main will catch it and handle it.

Reply:
updated